src/main/scala/de/frosner/ddq/constraints/ColumnColumnConstraint.scala

```diff
@@ -2,17 +2,42 @@ package de.frosner.ddq.constraints
 
 import org.apache.spark.sql.{Column, DataFrame}
 
+import scala.util.Try
+
 case class ColumnColumnConstraint(constraintColumn: Column) extends Constraint {
 
   val fun = (df: DataFrame) => {
-    val succeedingRows = df.filter(constraintColumn).count
-    val count = df.count
-    val failingRows = count - succeedingRows
+    val maybeSucceedingRows = Try(df.filter(constraintColumn).count)
+    val maybeCount = maybeSucceedingRows.map(_ => df.count)
+    val maybeFailingRows = maybeCount.map(count => count - maybeSucceedingRows.get)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/frank_7/drunken-data-quality/file/2305054507/issues/source?bid=3033737&fileBranchId=3196266#l12)